Buy/Graph.Client.swift

```diff
@@ -59,6 +59,9 @@ extension Graph {
     ///
     public class Client {
         
+        /// Cache policy to use for all `query` operations produced by this instance of `Graph.Client` 
+        public var cachePolicy: CachePolicy = .networkOnly
```

Comment:
Constructors should only require parameters that are essential for initialization, which a caching policy isn't. We can't predict what the use case might be so it's left as a mutable property. Is there any particular reason why you think this shouldn't be mutable?